spec/configuration_spec.rb

```diff
@@ -0,0 +1,27 @@
+require "spec_helper"
+require_relative "../lib/fake_idp/configuration"
+
+describe FakeIdp::Configuration do
+  it 'defaults encryption_enabled to false' do
+    expect(subject.encryption_enabled).to be_falsey
+  end
+
+  context 'when ENCRYPTION_ENABLED is set' do
+    before { ENV['ENCRYPTION_ENABLED'] = 'some_value' }
```

Comment:
One minor tweak here that will limit the spec cleanup and prevent any uncaught side-effects would be to stub the `ENV` object, e.g.

```ruby
allow(ENV).to receive(:[]).with('ENCRYPTION_ENABLED').and_return('some_value')
```

This fully encapsulates our setup to a single stub that is self-contained to the parent RSpec block, and negates the need for the `after` callbacks. Super minor preference, no change actually required.